service/src/main/java/bio/terra/pipelines/service/PipelineRunsService.java

```diff
@@ -148,78 +157,58 @@ private Map<String, Map<String, String>> prepareFileInputs(
 
     Map<String, Map<String, String>> fileInputsMap = new HashMap<>();
     for (String fileInputName : fileInputNames) {
+      String fileInputValue = userProvidedInputs.get(fileInputName).toString();
```

Comment:
i think its safer to do `(String) userProvidedInputs.get(fileInputName)` if you're looking to cast a value